Update enable-cloud-protection-microsoft-defender-antivirus.md (#10205)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus.md

1. Go to the Microsoft Endpoint Manager admin center ([https://endpoint.microsoft.com](https://endpoint.microsoft.com)) and sign in.

> 2. On the **Home** pane, select **Devices**> **Configuration Profiles**.

Reviewer: ```suggestion
2. On the **Home** pane, select **Devices** > **Configuration Profiles**.
```

3. Select the **Device restrictions** profile type you want to configure. If you need to create a new **Device restrictions** profile type, see [Configure device restriction settings in Microsoft Intune](/intune/device-restrictions-configure).

> 4. In the **Properties** section Click on **Edit** next to **Configuration settings** and click on **Microsoft Defender Antivirus**.

Reviewer: ```suggestion
4. In the **Properties** section, select **Edit** next to **Configuration settings** and then select **Microsoft Defender Antivirus**.
```
